### PR TITLE
[MERGE] call invitation cards

### PR DESCRIPTION
**Pull request by ThanhDodeurOdoo**

### CODE REVIEW

File: addons/mail/static/src/new/core/messaging_service.js [hunk 1]
```diff
@@ -375,6 +375,9 @@ export class Messaging {
                     }
                     break;
                 case "mail.record/insert":
+                    if (notif?.payload?.Thread?.rtcInvitingSession) {
```
Comment (ThanhDodeurOdoo) [inline addons/mail/static/src/new/core/messaging_service.js]: Looks like the new notification handler is not getting the notification sent there, even though the legacy code does:

https://github.com/odoo-dev/odoo/blob/master-discuss-refactoring-rtc-invitationCards2-tso/addons/mail/models/mail_channel_member.py#L232

Comment (ThanhDodeurOdoo) [reply addons/mail/static/src/new/core/messaging_service.js]: Whereas this notification is correctly received on both sides:

https://github.com/odoo-dev/odoo/blob/master-discuss-refactoring-rtc-invitationCards2-tso/addons/mail/models/mail_channel.py#L472

File: addons/mail/static/src/new/rtc/call.scss [hunk 2]
```diff
@@ -28,6 +28,26 @@
...
+    width: var(--width);
+    height: var(--height);
+    min-width: var(--width);
+    min-height: var(--height);
+    aspect-ratio: 16/9;
+}
+
+.o-mail-call-main-sidebarButton {
+    font-size: 1.2rem;
+    padding: map-get($spacers, 2);
```
Comment (alexkuhn) [inline addons/mail/static/src/new/rtc/call.scss]: I replaced this to `p-2`

File: addons/mail/static/src/new/rtc/call.scss [hunk 3]
```diff
@@ -28,6 +28,26 @@
     aspect-ratio: 16/9;
 }
 
+.o-mail-call-main {
+    min-width: 0;
+}
```
Comment (alexkuhn) [inline addons/mail/static/src/new/rtc/call.scss]: I replaced to `overflow-auto`

**Pull request merged**